snapserver/config.yaml

```diff
@@ -0,0 +1,47 @@
+---
+name: snapcast-server
+version: dev
+slug: snapcastserver
+description: Create a snapcast server
+url: https://github.com/Art-Ev/addon-snapserver
+codenotary: arthur@evrard.dev
+arch:
+  - aarch64
+  - amd64
+  - armv7
+  - i386
+host_network: true
+map: ["share:rw"]
+audio: true
+init: false
```

Comment:
Don't we need brackets here?

Reply:
> Don't we need brackets here?

I'm not sure (I'm new to HA) but based on official Spotify Connect addon I would say no : https://github.com/hassio-addons/addon-spotify-connect/blob/main/spotify/config.yaml